CODEOWNERS

```diff
@@ -1,5 +1,294 @@
+ext/hal/cmsis/*	maureen.helm@nxp.com    kumar.gala@linaro.org
+
+# BOARDS/ARC - ARDUINO 101 SSS
+boards/arc/arduino_101_sss/*	anas.nashif@intel.com
+
+# BOARDS/ARC - EM Starterkit
+boards/arc/em_starterkit/*	cjordan@synopsys.com
+
+# BOARDS/ARC - QUARK SE C1000 SS Devboard
+boards/arc/quark_se_c1000_ss_devboard/*	anas.nashif@intel.com
+
+# BOARDS/ARM - 96Boards CARBON
+boards/arm/96b_carbon/*	amit.kucheria@linaro.org    ricardo.salveti@linaro.org
+
+# BOARDS/ARM - 96Boards NITROGEN
+boards/arm/96b_nitrogen/*	amit.kucheria@linaro.org
+
+# BOARDS/ARM - ARDUINO 101 BLE
+boards/arm/arduino_101_ble/*	johan.hedberg@intel.com
+
+# BOARDS/ARM - CC32XX LAUNCHXL
+boards/arm/cc3200_launchxl/*	gil.pitney@linaro.org
+boards/arm/cc3220sf_launchxl/*	gil.pitney@linaro.org
+
+# BOARDS/ARM - NXP FRDM-K64F
+boards/arm/frdm_k64f/*	maureen.helm@nxp.com
+
+# BOARDS/ARM - NXP FRDM-KW41Z
+boards/arm/frdm_kw41z/*	maureen.helm@nxp.com
+
+# BOARDS/ARM - NXP Hexiwear
+boards/arm/hexiwear_k64/*	maureen.helm@nxp.com
+
+# BOARDS/ARM - NORDIC NRF51 REDBEAR BLENANO
+boards/arm/nrf51_blenano/*	ricardo.salveti@linaro.org
+
+# BOARDS/ARM - NORDIC NRF52 PCA10040
+boards/arm/nrf52_pca10040/*	carles.cufi@nordicsemi.no
+
+# BOARDS/ARM - NUCLEO-64 F401RE Devboard
+boards/arm/nucleo_f401re/*	amit.kucheria@linaro.org    ricardo.salveti@linaro.org
+
+# BOARDS/ARM - ARM LTD V2M Beetle
+boards/arm/v2m_beetle/*	vincenzo.frascino@linaro.org
+
+# BOARDS/NIOS2 - ALTERA MAX10
+boards/nios2/altera_max10/*	andrew.p.boie@intel.com
+
+# BOARDS/X86 - ARDUINO 101
+boards/x86/arduino_101/*	anas.nashif@intel.com
+
+# BOARDS/X86 - Galileo
+boards/x86/galileo/*	anas.nashif@intel.com
+
+# BOARDS/X86 - QUARK D2000 Devboard
+boards/x86/quark_d2000/*	anas.nashif@intel.com
+
+# BOARDS/X86 - QUARK SE C1000 Devboard
+boards/x86/quark_se_c1000/*	anas.nashif@intel.com
+
+# BLUETOOTH
+subsys/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+include/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+include/drivers/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+drivers/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+samples/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+tests/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+doc/subsystems/bluetooth/*	johan.hedberg@intel.com    luiz.dentz@gmail.com    szymon.janc@gmail.com
+
+# BLUETOOTH CONTROLLER
+subsys/bluetooth/controller/*	vinayak.kariappa.chettimada@nordicsemi.no    carles.cufi@nordicsemi.no
+
+# CC32XX SDK
+ext/hal/ti/cc3200sdk/*	gil.pitney@linaro.org
+ext/hal/ti/cc3220sdk/*	gil.pitney@linaro.org
+
+# CC32XX SOC - TI SIMPLELINK
+arch/arm/soc/ti_simplelink/cc32xx	gil.pitney@linaro.org
+
+# DOCUMENTATION
+doc/*	david.b.kinder@intel.com    inaky.perez-gonzalez@intel.com
+*.rst	david.b.kinder@intel.com    inaky.perez-gonzalez@intel.com
```

Comment:
This file is converted automatically from the `MAINTAINERS` file with that script.  I'll add a comment in the first few lines of this file stating that.